WalletWasabi/Nostr/NostrExtensions.cs

```diff
@@ -89,6 +89,7 @@ void ConfigureSocket(WebSocket socket)
 
 		await client.ConnectAndWaitUntilConnected(cancellationToken).ConfigureAwait(false);
 		await client.SendEventsAndWaitUntilReceived(evts, cancellationToken).ConfigureAwait(false);
+		await client.Disconnect().ConfigureAwait(false);
 	}
```

Comment:
What happens if the cancellationToken fires and the code go through? Works fine?